reporters/reporter-default/src/main/java/org/radargun/reporting/commons/DataReporter.java

```diff
@@ -0,0 +1,175 @@
+package org.radargun.reporting.commons;
+
+import java.io.File;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeSet;
+
+import org.radargun.config.Cluster;
+import org.radargun.logging.Log;
+import org.radargun.logging.LogFactory;
+import org.radargun.reporting.Report;
+import org.radargun.stats.Statistics;
+import org.radargun.stats.representation.DataThroughput;
+import org.radargun.stats.representation.DefaultOutcome;
+import org.radargun.stats.representation.MeanAndDev;
+import org.radargun.stats.representation.OperationThroughput;
+import org.radargun.stats.representation.Percentile;
+import org.radargun.utils.Utils;
+
+public class DataReporter {
```

Comment:
Some class JavaDoc here would be good! I can see this class is used by the `CsvReporter` class below, so there must be something good this class is doing! :-)